guide/kh/index.md

```diff
@@ -0,0 +1,5 @@
+---
+section: guide
+lang: KH
+title: Khmer
+---
```

Comment:
Hi Morchickit!
I'm very new to GitHub. Was working on OD Handbook v.1 on Transifex.com.
We are going to run the last session to finalize the KHM translation - see details on our FB event page (https://www.facebook.com/events/366189123590251/).

And for the very start, I already made a mistake w/ the language code - please help rename the folder to KM (KH is the country domain, while KM is for the language).
Thank you.